.github/workflows/record_snapshots.yml

```diff
@@ -135,11 +137,26 @@ jobs:
           git config --local user.name "github-actions[bot]"
           git pull
           git add app/screenshots/*
-          git commit -m "Updated screenshots"
+          git commit -m "Updated snapshots for '${{ matrix.variant }}'"
 
       - name: Push changes
         if: steps.check.outputs.triggered == 'true'
         uses: ad-m/github-push-action@master
         with:
           branch: ${{ fromJSON(steps.get-pr.outputs.result).head.ref }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Notify PR of snapshots success
```

Comment:
It's expected that some runs will fail. For instance, the tests were changed to affect only DM screenshots. In all the other runs it will fail as there are no changes to commit.

Reply:
Lets get it merged and tested first

Reply:
That is ok, but we can batch the message at the end of the matrix execution. IT will be better than getting spammed with however many messages per variant